sim/dex-data.js

```diff
@@ -214,8 +213,8 @@ class Format extends Effect {
 		 */
 		this.unbanlist = this.unbanlist || [];
 		/**
-		 * List of ruleset and banlist changes in a custom format.
-		 * @type {?string[]}
+		 * List of validated ruleset and banlist changes in a custom format.
+		 * @type {?[string, string, number, string[]][]}
```

Comment:
This is really not how I want this to be stored. I'm going to code around half of this myself, I guess.